strr-web/components/bcros/form-section/property/Address.vue

```diff
@@ -22,59 +22,118 @@
         </UFormGroup>
       </div>
       <div class="flex flex-row justify-between w-full mb-[40px] mobile:mb-[16px]">
-        <UFormGroup name="Address" class="d:pr-[16px] flex-grow">
+        <UFormGroup
+          name="streetNumber"
+          class="d:pr-[16px] flex-grow"
+          :error="streetNumberError"
+        >
+          <UInput
+            :id="streetNumberId"
+            v-model="streetNumber"
+            :aria-label="t('createAccount.contactForm.streetNumber')"
+            :placeholder="t('createAccount.contactForm.streetNumber')"
+            @input="
+              onAddressInput(false); emit('resetFieldError', [
+                'streetNumber', 'streetName', 'city', 'province', 'postalCode'
+              ])
+            "
+            @click="addressComplete(false)"
+            @blur="emit('validateStreetNumber')"
+            @change="emit('validateStreetNumber')"
+          />
+        </UFormGroup>
+        <UFormGroup
+          name="streetName"
+          class="d:pr-[16px] flex-grow"
+          :error="streetNameError"
+        >
+          <UInput
+            :id="streetNameId"
+            v-model="streetName"
+            :aria-label="t('createAccount.contactForm.streetName')"
+            :placeholder="t('createAccount.contactForm.streetName')"
+            @input="
+              onAddressInput(true); emit('resetFieldError', [
+                'streetNumber', 'streetName', 'city', 'province', 'postalCode'
+              ])
+            "
+            @click="addressComplete(true)"
+            @blur="emit('validateStreetName')"
+            @change="emit('validateStreetName')"
+          />
+        </UFormGroup>
+        <UFormGroup
+          name="unitNumber"
+          class="d:pr-[16px] flex-grow"
+          :error="unitNumberError"
+        >
           <UInput
-            :id="id"
-            v-model="address"
-            :placeholder="t('createAccount.contactForm.address')"
-            aria-label="address"
-            @input="onAddressInput"
-            @click="addressComplete()"
+            v-model="unitNumber"
+            :aria-label="t('createAccount.contactForm.unitNumberRequired')"
+            :placeholder="unitNumberPlaceholder"
+            @input="emit('resetFieldError', ['unitNumber'])"
```

Comment:
I noticed that this is different from earlier implementation, where `resetFieldError` was emitting a string (and not array). Something changed, you want to reset multiple fields at once?

Reply:
If this is what the requirement is, I'm good with it, thanks for clrifying.